### PR TITLE
Add a guide explaining how to create a sample PR

### PR DESCRIPTION
**Pull request by GuptaManan100**
This PR adds a contributor guide explaining how to fix an issue in Vitess and how to create a PR for it complete with the unit tests. The sample issue chosen for this is https://github.com/vitessio/vitess/issues/4069 and the final sample PR is https://github.com/vitessio/vitess/pull/9456.

#872

### CODE REVIEW

**Review by frouioui — APPROVED** [review]
This is nice!

File: content/en/docs/13.0/contributing/reference/sample-first-issue.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+After having Vitess locally setup for development, you can go ahead and choose an issue to work on from [here](https://github.com/vitessio/vitess/issues).
+For first time contributors, some of the issues have been marked with the label *Good First Issue*. These can be found [here](https://github.com/vitessio/vitess/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+First+Issue%22).  
+For this walkthrough, we will pick issue [#4069](https://github.com/vitessio/vitess/issues/4069).
+Let us dive right in!
+
+The issue is that Vitess's parser does not support parsing a string literal that starts with a number. 
+The first thing to do is to reproduce the error as a unit test.
+Since it is a parsing error, we can add a unit test for it in the [parse_test.go](https://github.com/vitessio/vitess/blob/main/go/vt/sqlparser/parse_test.go) file. More information on how to contribute to the AST parser is available [here](../contributing-to-ast-parser).  
+We add the test to the already existing test **TestValid** by adding an additional test case to the *validSQL* list -
```
Comment (deepthi) [inline content/en/docs/13.0/contributing/reference/sample-first-issue.md]:
```suggestion
We can add this to **TestValid** by adding an additional test case to the *validSQL* list -
```

File: content/en/docs/13.0/contributing/reference/sample-first-issue.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+			expectedOut := tcase.out
+			if expectedOut == "" {
+				expectedOut = tcase.in
+			}
+			require.Equal(t, expectedOut, out)
+		})
+	}
+}
+```
+We can commit this change, and now focus on fixing the issue.
```
Comment (deepthi) [inline content/en/docs/13.0/contributing/reference/sample-first-issue.md]: Amend the previous commit?

File: content/en/docs/13.0/contributing/reference/sample-first-issue.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+	}
+	return ID, tkn.buf[start:tkn.Pos]
+}
+```
+We can now verify that our added unit test in *token_test.go* works perfectly and can commit this change.
+
+As a final step, we run the *parse_test.go* file to ensure that everything works. We fix any tests whose expectations have changed or any incorrect tests to reflect the change and commit it.
+
+With these changes, the issue is resolved! But our work is not yet complete...  
+We need to create a pull request for our changes and request for reviews.
```
Comment (deepthi) [inline content/en/docs/13.0/contributing/reference/sample-first-issue.md]: Do new contributors have access to request reviews? I don't think so.

**Review by deepthi — APPROVED** [review]
💯 
Can you address the comments? Then we can merge this.

File: content/en/docs/13.0/contributing/reference/sample-first-issue.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/13.0/contributing/reference/sample-first-issue.md]: fixed

File: content/en/docs/13.0/contributing/reference/sample-first-issue.md (hunk 2 above)
Comment (GuptaManan100) [reply content/en/docs/13.0/contributing/reference/sample-first-issue.md]: Fixed

File: content/en/docs/13.0/contributing/reference/sample-first-issue.md (hunk 3 above)
Comment (GuptaManan100) [reply content/en/docs/13.0/contributing/reference/sample-first-issue.md]: fixed

**Pull request merged**